src/cev/_embedding_widget.py

```diff
@@ -65,7 +65,7 @@ def _validate_labels(self, proposal: object):
         # convert to category if not already
         return (
             proposal.value
-            if proposal.value == "category"
```

Comment:
wait, shouldn't this be:

```suggestion
            if proposal.value.dtype == "category"
```